application/helpers/update/updatedb_helper.php

```diff
@@ -3084,6 +3084,31 @@ function db_upgrade_all($iOldDBVersion, $bSilent = false)
             $oDB->createCommand()->update('{{settings_global}}', array('stg_value' => 426), "stg_name='DBVersion'");
             $oTransaction->commit();
         }
+
+        /**
+         * Add missing noTablesOnMobile.css to vanilla and bootswatch configs
+         */
+        if ($iOldDBVersion < 427) {
+            $oTransaction = $oDB->beginTransaction();
+            // Update vanilla config
+            $oDB->createCommand()->update(
+                '{{template_configuration}}',
+                [
+                    'files_css' => '{"add":["css/ajaxify.css","css/theme.css","css/custom.css","css/noTablesOnMobile.css"]}',
```

Comment:
If someone update himself a template_configuration in DB to add or remove a css : you broke it …

You must
1. Read current
2. decode current 
3. Leave inherit as inherit
4. Add `css/noTablesOnMobile.css` 
5. Save it

Reply:
These changes only apply for vanilla and bootswatch. I think template_configuration can't be changed for those. right?

Reply:
Reviewed. No in sync with config.xml

Reply:
It can be manually edited (master screenshot)
![Capture d’écran du 2020-07-09 08-37-36](https://user-images.githubusercontent.com/1439428/87005583-82776b00-c1bf-11ea-863c-cbad2ccb025f.png)

Reply:
> It can be manually edited (master screenshot)
> ![Capture d’écran du 2020-07-09 08-37-36](https://user-images.githubusercontent.com/1439428/87005583-82776b00-c1bf-11ea-863c-cbad2ccb025f.png)

Honestly, I couldn't find that screen in v4.3.3
Let me take another look.

Reply:
I have checked with v4.3.3 and didn't found that screen.
Let me triple check and be back with you

Reply:
Oh right , but can be updated in 3.X.

Your right : it's a 3.X screenshit (think i took it on master sorry).

Still : dislike the fix but not the fix : the global way for updating a theme …